hack/e2e/environment.sh

```diff
@@ -21,6 +21,24 @@ function os_check() {
 export CAPI_VERSION=${CAPI_VERSION:-"v1beta1"}
 export CAPM3_VERSION=${CAPM3_VERSION:-"v1beta1"}
 export NUM_NODES=${NUM_NODES:-"4"}
+export UPGRADE_TEST=${UPGRADE_TEST:-false}
+if $UPGRADE_TEST; then
+    export CAPI_VERSION="v1alpha4"
+    export CAPM3_VERSION="v1alpha5"
+fi
+
+# Override project infra vars that point to 
+# the current branch to build capm3 image and crds
+if [[ "${CAPM3_VERSION}" == "v1alpha5" ]]; then 
+    export CAPM3RELEASE="v0.5.5"
+    export CAPIRELEASE="v0.4.7"
+    export CAPM3BRANCH="release-0.5"
```

Comment:
These are not needed to be set at all, see why:

- CAPM3RELEASE is set to the [latest](https://github.com/metal3-io/metal3-dev-env/blob/master/lib/releases.sh#L47) release for v1a5 API of CAPM3 when you export it on line 27
- Same for CAPI release, it is set to  is set to the [latest](https://github.com/metal3-io/metal3-dev-env/blob/master/lib/releases.sh#L37) release for v1a4 API of CAPI when you export it on line 26
- CAPM3BRANCH is [set](https://github.com/metal3-io/metal3-dev-env/blob/master/lib/common.sh#L146) accordingly based on the CAPM3RELEASE

Reply:
This is hard coded to a fixed release because on the e2e config file I am still had coding release just this that will be conflict between v1beta1 and v1alpha5 release if we parameterize them we can work on parameterize that separately

Reply:
Okay, I was suggesting to remove from the file not to parameterize it

Reply:
once the config file will be parameterized we can remove them to use always the latest but for now better to keep this fixed here